templates/repo/commits.tmpl

```diff
@@ -5,14 +5,12 @@
 	  <div class="ui secondary menu">
 	    {{template "repo/branch_dropdown" .}}
 	    <div class="fitted item">
-	      <div class="ui basic small button">
```

Comment:
I mean, you removed the head `<div>` but not the tail `</div>`.

I think it's okay to remove, though.